cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -268,6 +270,45 @@ public Network convert(ReportNode reportNode) {
         return network;
     }
 
+    /**
+     * Retrieve the Collection of OperationalLimitGroups for identifiable that have flow limits (branch, dangling line, 3w-transformer).
+     * If the collection has only one element, it gets to be the identifiable's selectedGroup.
+     * If there is more than one element in the collection, do nothing. The identifiable's selectedGroup should then be set by the user after the conversion.
+     * @param context The conversion's Context.
+     */
+    private void setSelectedOperationalLimitsGroup(Context context) {
```

Comment:
I think that to avoid all these loops we could do the selection in `LoadingLimitsMapping::addAll`.